src/harness/unittests/session.ts

```diff
@@ -16,8 +16,8 @@ namespace ts.server {
         directoryExists: () => false,
         getDirectories: () => [],
         createDirectory: noop,
-        getExecutingFilePath(): string { return void 0; },
-        getCurrentDirectory(): string { return void 0; },
+        getExecutingFilePath(): string { return ""; },
+        getCurrentDirectory(): string { return ""; },

```

Comment:
Deliberate?

Reply:
Yes

Reply:
Clarification: This is what it does in `master`

Reply:
Right... but is it in any way related to the fixes happening here? If not, is there any reason to need this change also here?

Reply:
Actually.... it's in the harness, not the product, so if the tests are passing I don't really care :-p

Reply:
We need this because we compute the default safe list location based on the current directory / current executing file path